Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -511,21 +537,20 @@ - (NSMutableDictionary *)adOptionsForServerParameters:(NSDictionary<NSString *,
     if ( [serverParameters al_containsValueForKey: @"is_muted"] ) // Introduced in 6.10.0
     {
         BOOL muted = [serverParameters al_numberForKey: @"is_muted"].boolValue;
-        [VungleSDK sharedSDK].muted = muted;
-        
-        options[VunglePlayAdOptionKeyStartMuted] = @(muted);
+        // Overwritten by `mute_state` setting for Vungle only if YES,
+        // Publisher has to set change the value for SetMuted to NO at Vungle dashboard level for playing an ad with sound.
```

Comment:
This comment is weirdly phrased.

// The mute status will only be overwritten if YES.
// If the publisher wants the ad to play with sound, the publisher should not set this property.